pkgs/development/libraries/libpcap/default.nix

```diff
@@ -25,6 +25,8 @@ stdenv.mkDerivation rec {
 
   prePatch = stdenv.lib.optionalString stdenv.isDarwin ''
     substituteInPlace configure --replace " -arch i386" ""
+  '' + ''
+    sed -i '1i#include <limits.h>' pcap-usb-linux.c
```

Comment:
Make sure to reference this pr: https://github.com/the-tcpdump-group/libpcap/pull/735

This shouldn't be needed once the next release happens.

Reply:
Maybe even https://github.com/the-tcpdump-group/libpcap/commit/aafa3512b7b742f5e66a5543e41974cc5e7eebfa.patch, with sed calls it is hard to know when to remove it.

Reply:
Didn't know there was a PR, found fix in void and figured local sed was clearer than patch from void for this; using upstream patch and mentioning PR is definitely the way to go.  Good calls!